test/node/bson_test.js

```diff
@@ -2268,4 +2269,117 @@ describe('BSON', function () {
     expect(() => BSON.serialize(badArray)).to.throw();
     expect(() => BSON.serialize(badMap)).to.throw();
   });
+
+  describe('Should support util.inspect for', function () {
+    /**
+     * @ignore
+     */
+    it('Binary', function (done) {
+      const binary = new Binary(Buffer.from('0123456789abcdef0123456789abcdef', 'hex'), 4);
+      expect(inspect(binary)).to.equal('Binary("0123456789abcdef0123456789abcdef", 4)');
+      done();
```

Comment:
nit: you only need `done();` when you have to run a callback so you can drop the done calls but that's optional.

Reply:
Right... was a little confused as I've found some other sync tests in there using the same pattern and wasn't that sure anymore 🤔 Sorry about that!